runtime/picasso/src/lib.rs

```diff
@@ -104,9 +104,9 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	//   runtime in substitute for the on-chain Wasm runtime unless all of `spec_name`,
 	//   `spec_version`, and `authoring_version` are the same between Wasm and native.
 	spec_version: 1000,
```

Comment:
Don't we increase spec version here?

Reply:
picasso spec version is currently 200 on kusama